test/e2e/gorillamux/traces.json

```diff
@@ -28,21 +28,22 @@
                 {
                   "key": "http.method",
                   "value": {
-                    "stringValue": ""
+                    "stringValue": "GET"
                   }
                 },
                 {
                   "key": "http.target",
                   "value": {
-                    "stringValue": ""
+                    "stringValue": "/users/foo"
                   }
                 }
               ],
               "kind": 2,
+              "name": "/users/foo",
               "parentSpanId": "",
-              "spanId": "",
+              "spanId": "xxxxx",
               "status": {},
-              "traceId": ""
+              "traceId": "xxxxx"
```

Comment:
interesting that now, instead of being empty, this appears to be a duplicate of the span that's already here (below this one). Tough to tell if it's actually duplicated because of the normalized trace/span ids

Reply:
If I expand the view of the changes to this file into unchanged territory, I notice that the two spans in this trace are in 2 different scopes: one is `github.com/gorilla/mux` and one is `net/http`.

Reply:
@robbkidd ahh that's cool, thanks for explaining